### ISSUE TITLE
Real getters/setters instead of properties.

### ISSUE THREAD

**Issue by vbence** [TiVo/intellij-haxe#164]
I am using *real* getters and setters like: `getField` or `setField` which are called as functions (instead of *properies* which are accessed as fields).

Until the last version it was easy, just had to remove the extra elements from variable declarations (the `(get,set)` part):

    private var field(get, set):String;

In the last version getters and setters (propery accessors) are generated in the `get_field` format which can not be easily converted to camel case.

Can you please provide the means to generate (classic) getters and setters without too much hassle? Thank you.

**Comment by as3boyan** [comment]
http://old.haxe.org/manual/haxe3/migration#property-accessors

**Comment by vbence** [comment]
@as3boyan Yep I know about that, it is a great feature, I imagine, to people using **properties**. I'm simply using classic *getter and setter functions*, which are called as functions. - I'm not accessing them as `myObject.myField`, but `myObject.getMyField()`.

Properties are another programming style, there is nothing wrong with them, but I'm vouching for getters and setters. My personal reasons: I work in a multi-language environment, and this style is much more portable (function exists in all of my target languages - js, php, java - but properties as a concept do not).

**Comment by as3boyan** [comment]
Perhaps we can add it. What would you like to see in function body?

get:
return value;

set
return value = _value;

same as now?

**Comment by vbence** [comment]
Yep, current function bodies are perfect as they are.

    public function setId(value:String) {
        return this.id = value;
    }

    public function getId():String {
        return id;
    }

Only *camelCase function names* are amiss.

There was a radical change in the last release handling the variables too. Currently the following is forced by the code:

    @:isVar public var id(get, set):String;

Which:
 * Makes the member public
 * Removes all meta previously added to the member
 * Adds @:isVar
 * Adds the `(get, set)` clause to the declaration.

When you are using classic getters/setters, there is no need to do any transformation on the declaration part. As the variable already exists when someone is using this code insertion, it is safe to assume they already set the visibility to their liking.

Thanks a lot for your openness and taking your time with this issue :)
